libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -330,9 +330,9 @@ AC_PosControl::AC_PosControl(AP_AHRS_View& ahrs, const AP_InertialNav& inav,
 
 /// input_pos_xyz - calculate a jerk limited path from the current position, velocity and acceleration to an input position.
 ///     The function takes the current position, velocity, and acceleration and calculates the required jerk limited adjustment to the acceleration for the next time dt.
-///     The kinematic path is constrained by the maximum acceleration and time constant set using the function set_max_speed_accel_xy and time constant.
-///     The time constant defines the acceleration error decay in the kinematic path as the system approaches constant acceleration.
-///     The time constant also defines the time taken to achieve the maximum acceleration.
+///     The kinematic path is constrained by the maximum acceleration and jerk set using the function set_max_speed_accel_xy.
```

Comment:
When I read this comment it makes me think that set_max_speed_accel_xy() should have a "jerk" argument but it doesn't seem to.

Reply:
I think I cleared this one up.